components/list/list-item-mixin.js

```diff
@@ -311,6 +317,14 @@ export const ListItemMixin = superclass => class extends composeMixins(
 				margin: 0;
 			}
 
+			:host([color]) [slot="outside-control-container"] {
+				margin-left: -6px;
```

Comment:
This changes things a bit from current behaviour, which is `-12px` margin-left, which shows up on hover. The last 2 items in the list below really show the difference. If we add the ability for siblings to leave space when a sibling has a color then the difference will no longer exist, but it is a change in behaviour. Any thoughts on this? I'll be checking with design as well.

<img width="750" alt="Screenshot 2023-08-24 at 1 11 13 PM" src="https://github.com/BrightspaceUI/core/assets/6804600/76e36afb-74c5-47f5-8f50-70f737eb3a2b">

Reply:
Are you sure you'd want to override the CSS immediately above (`margin: 0`) for the draggable case? Still refreshing my memory as to what this margin is for. 😬

Reply:
The design specifies 6 px between the color and left border on hover. If we don't have this then it looks like this:
<img width="609" alt="Screenshot 2023-08-28 at 10 59 12 AM" src="https://github.com/BrightspaceUI/core/assets/6804600/244c617f-d5f6-428d-af3e-cc62003d727c">

Reply:
Gotcha. What about the draggable case?

Reply:
Ah right this does add extra to the left in that case. Will fix